recipes/gsl/meta.yaml

```diff
@@ -0,0 +1,24 @@
+package:
```

Comment:
Do you need gsl 2.2? gsl is available in conda-forge and I think the correct way would be to upgrade this version.